e2e/pages/adf/contentServicesPage.ts

```diff
@@ -259,6 +254,7 @@ export class ContentServicesPage {
     createNewFolder(folder) {
         this.clickOnCreateNewFolder();
         this.createFolderDialog.addFolderName(folder);
+        browser.sleep(500);
```

Comment:
not a good practice to use browser.sleep. Is this really needed? can't be avoided by using browser.wait for a specific element or something?